core/tee/entry_std.c

```diff
@@ -599,8 +601,6 @@ static TEE_Result default_mobj_init(void)
 
 #ifdef CFG_SECURE_DATA_PATH
 	sdp_mem_mobjs = core_sdp_mem_create_mobjs();
-	if (!sdp_mem_mobjs)
```

Comment:
Wouldn't this still be an error?

Reply:
yes, as we must have a SDP mobj table if CFG_SECURE_DATA_PATH is defined.

Reply:
done here 2e3ba1b18cd2138acba8071f1617fc18a5ac0329